media-source-respec.html

```diff
@@ -2719,6 +2711,280 @@ <h3>Event Summary</h3>
       </section>
     </section>
 
+    <section data-cite="mimesniff" data-dfn-for="ManagedMediaSource">
+      <h2><dfn>ManagedMediaSource</dfn> interface</h2>
+      <p>
+        A {{ManagedMediaSource}} is a {{MediaSource}} that actively manages its
+        memory content. Unlike a {{MediaSource}}, the [=user agent=] can evict content
+        through the [=ManagedMediaSource/memory
+        cleanup=] algorithm from its {{MediaSource/sourceBuffers}} (populated with {{ManagedSourceBuffer}})
+        for any reason.
```

Comment:
It's not unusual to only say what should happen in spec, and not what should not happen (because most thing shouldn't happen). But this would be nice in a non-normative note (this section is normative).

Reply:
I wrote a note below to address this https://github.com/w3c/media-source/pull/329/files#r1414960629

Reply:
See aside below...